jena-arq/src/test/java/org/apache/jena/sparql/algebra/optimize/TestOptDistinctReduced.java

```diff
@@ -218,7 +218,7 @@
         testQuery(queryString, tOrderByDistinctApplication, opExpectedString) ;
     }
     
-    @Test public void distinct_order_by_application_03()
+    /*@Test*/ public void distinct_order_by_application_03_old()
```

Comment:
If this test case is no longer valid why not simply delete it?

Reply:
Done. I meant to but missed coming back to this while juggling the other pull requests.

Comment added to 02.